packages/circuit-ui/components/Button/Button.tsx

```diff
@@ -75,10 +74,6 @@ export interface BaseProps {
    * Function that's called when the button is clicked.
    */
   'onClick'?: (event: ClickEvent) => void;
-  /**
-   * Additional data that is dispatched with the tracking event.
-   */
-  'tracking'?: TrackingProps;
```

Comment:
We'll need to update `next` from `main`, because in the latest `main` this is marked as deprecated (so there will be conflicts)

https://github.com/sumup-oss/circuit-ui/blob/8ab8c2098967d04634e9a6f47830c54bee603e99/packages/circuit-ui/components/Button/Button.tsx#L79

Reply:
Yup, I intend to update `next` and then rebase this PR once we get closer to the v7 release.